src/components/dashboard/MicroIcon/QRIcon.jsx

```diff
@@ -69,12 +69,11 @@ export default class QRIcon extends PureComponent {
 
         </span>
         <Popover
-          zDepth={3}
           open={this.state.isQROpen}
           anchorEl={this.state.qrAnchorEl}
           anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
-          targetOrigin={{ horizontal: 'right', vertical: 'top' }}
-          onRequestClose={() => this.handleQRClose()}
+          onClose={() => this.handleQRClose()}
```

Comment:
It is better to not use fat-arrow functions in JSX.

Constructor:
```
  constructor (props) {
    super(props)
    this.state = {
      isQROpen: false,
      qrData: null,
      qrAnchorEl: null,
    }
    this.handleQRClose = this.handleQRClose.bind(this)
  }
```

And here:
```
onClose={this.handleQRClose}
```